src/content/developers/docs/development-networks/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Development Networks
+description: An overview of development networks and the tools available to help build Ethereum applications.
+lang: en
+sidebar: true
+---
+
+When building an Ethereum application, you'll want to run it on a local network to see how it works before deploying it.
+
+Similar to how you might run a local server on your computer for web development, you can create a local blockchain instance to test your dapp. This allows for much faster iteration than a public testnet.
```

Comment:
You can also mention here that in these networks you don't need to deal with ETH, as that's something that many people new to Ethereum don't automatically get.